docroot/themes/custom/yukonca_glider/src/sass/base/_main.scss

```diff
@@ -62,6 +62,57 @@ h1#heading {
     &.region-content {
       @apply pt-3.75;
     }
+
+    // Start Sidebar design
+
+    &.region-sidebar-first {
+      .block-menu {
+        .block-title {
+          @apply font-medium pt-[15px] pb-[26.5px] mt-[15px] text-lg leading-1.1;
+        }
+        
+        > .menu {
+          @apply ml-0;
+          
+          > .menu-item {
+            @apply pt-0 list-none;
+
+            [data-drupal-link-system-path] {
+              @apply py-[15px] px-[10px] inline-block w-full;
+            }
+
+            &::marker {
+              @apply hidden;
+            }
+
+            &:nth-child(odd) {
+              > [data-drupal-link-system-path] {
+                @apply bg-[#F1F1F1];
```

Comment:
Please try to avoid using colors directly.
All colors should be listed in the tailwind configs `docroot/themes/custom/yukonca_glider/tailwind.config.js`
this one is already defined, so replace to `@apply bg-gray-light;` please